developer/testing/release-process.md

```diff
@@ -70,6 +70,13 @@ Notes:
 - Wait for all `master` tests to pass.
 - Follow the format of previous release, copy change log from release PR into the release notes.
 
-## Cleanup
+## Post-release checklist: All releases
 
-- Review issues that the release resolves, that they are closed and stale branches removed.
+- Core team: Review issues that the release resolves, that they are closed and stale branches removed.
+- Community team: Update Gitter with links to the latest release, changelog and update instructions.
+
+## Post-release checklist: Major releases
+
+- Community team: Tag a new Documentation release and update Editorial about any new documentation tutorial links, new FAQ questions and other relevant content for social media channels.
```

Comment:
Should we be more explicit here about the TODO's? LIke Tweet, Facebook, etc?

Reply:
I don't think we need to be explicit here b/c/ it's not up to the Devs/Comm Eng team what content gets published where. It's up to the Editorial Manager to decide what media channels are the most appropriate for the content (email newsletter v. social media v. whatever else is next). The most important thing from our side is that we get the links to them.